### PR TITLE
[INFRA-351] enable ML2/OVN and DVR by default

### PR DESCRIPTION
**Pull request by g0rgamesh**
Dependent on: https://review.opendev.org/c/openstack/kolla-ansible/+/896534
Merged.

### CODE REVIEW

**Comment by markgoddard** [comment]
This branch needs to be rebased onto stackhpc/zed.

File: releasenotes/notes/enable_ovn-b84974f398242dcb.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Enabled ML2/OVN by default as checks preventing accidental migration
+    from ML2/OVS were added in kolla-ansible.
```
Comment (markgoddard) [inline releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: If you are using a Neutron plugin other than ML2/OVN, set `kolla_enable_ovn` to `false`.

Comment (markgoddard) [reply releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: Also point out that OVN distributed FIP is enabled, and how to disable it if necessary.

Comment (g0rgamesh) [reply releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: Done.

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml [hunk 2]
```diff
@@ -23,7 +23,6 @@ kolla_enable_mariabackup: true
 kolla_enable_neutron_sriov: true
 kolla_enable_octavia: true
 kolla_enable_opensearch: true
-kolla_enable_ovn: true
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-builder/stackhpc-ci.yml:26]: Please remove this from the other files I mentioned in the original PR

File: releasenotes/notes/enable_ovn-b84974f398242dcb.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    Enabled ML2/OVN by default as checks preventing accidental migration
+    from ML2/OVS were added in kolla-ansible. If you are using a Neutron
+    plugin other than ML2/OVN, set `kolla_enable_ovn` to `false`.
+    OVN distributed FIP is also enabled, to disable it set
+    `neutron_ovn_distributed_fip` to `false`.
```
Comment (markgoddard) [inline releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]:
```suggestion
    `neutron_ovn_distributed_fip` to `false` in `etc/kayobe/kolla/globals.yml`.
```

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml (hunk 2 above)
Comment (g0rgamesh) [reply etc/kayobe/environments/ci-builder/stackhpc-ci.yml:26]: Done.

**Review by markgoddard — APPROVED** [review]
LGTM

**Comment by g0rgamesh** [comment]
> LGTM

Thanks, I'll merge once ovs check for Zed is merged upstream.

**Pull request merged**